tests/bud/buildkit-mount-from/Dockerfilecachemultiplefrom

```diff
@@ -0,0 +1,10 @@
+FROM alpine as builder
+COPY hello .
+
+FROM alpine as builder2
+COPY hello2 .
+
+FROM alpine
+RUN mkdir /test
+# use another image as cache source
```

Comment:
Suggest "other stages" instead of "another image".

Reply:
fixed